module/Activity/src/Activity/Service/Signup.php

```diff
@@ -87,10 +87,14 @@ public function getSignedUpData(ActivityModel $activity)
         }
 
         $fieldValueMapper = $this->getServiceManager()->get('activity_mapper_activity_field_value');
+        $memberMapper = $this->getServiceManager()->get('decision_mapper_member');
         $result = [];
         foreach($activity->getSignUps() as $signup){
             $entry = [];
-            $entry['member'] = $signup->getUser()->getMember()->getFullName();
+            //$signup->getUser()->getMember() yields an empty result if the signup is new.
+            //(i.e. a signup has taken place in the same request)
+            $member = $memberMapper->findByLidnr($signup->getUser()->getLidnr());
```

Comment:
Since `$signup->getUser()->getMember()` gets a doctrine proxy object (`DoctrineORMModule\Proxy\__CG__\Decision\Model\Member`), filled with null-values if the signup has been created in the same request, I have used a different way to get the name. I don't know enough about Doctrine to either tell if this a symptom of another bug or how to properly do this. This way does seem to work, though.